projectq/backends/_printer_test.py

```diff
@@ -19,8 +19,12 @@
 import pytest
 
 from projectq import MainEngine
-from projectq.cengines import DummyEngine, InstructionFilter
-from projectq.ops import T, H, Command, NOT, Measure
+from projectq.cengines import (DummyEngine,
+                               InstructionFilter,
+                               LogicalQubitIDTag,
```

Comment:
Can we add this somewhere else? E.g., types (qubit)?

Reply:
Went into `projectq.meta`